src/api/weatherAPI.ts

```diff
@@ -0,0 +1,56 @@
+import axios, {AxiosResponse} from "axios";
+import {apiKEY} from "../apiKey.env";
+
+export async function autoLocationWeatherDisplay(longitude: number, latitude: number) {
+    try {
+        console.log(longitude, latitude)
+        const response: AxiosResponse = await axios.get(`https://api.openweathermap.org/data/2.5/weather?&lat=${latitude}&lon=${longitude}&units=metric&appid=${apiKEY}&lang=ru`)
+        return responseHandler(response)
+    }
+    catch (err) {
+        console.log(`Ошибка:${err}`)
+    }
+}
+
+export async function getWeatherToday(cityTitleValue: string) {
+    try {
+        const response:AxiosResponse = await axios.get(`https://api.openweathermap.org/data/2.5/weather?q=${cityTitleValue}&units=metric&appid=${apiKEY}&lang=ru`)
+            return responseHandler(response)
+    } catch (err) {
+        console.log(`Ошикба:${err}`)
+    }
+}
+
+function responseHandler(response?: AxiosResponse | undefined) {
```

Comment:
in typescript you don't write response?: AxiosResponse | undefined. It's either response?: AxiosResponse, or response: AxiosResponse | undefined (the first one is preferred)